### ISSUE TITLE
Classification results

### ISSUE THREAD

**Issue by sheshap** [j1a0m0e4sNTU/3dgcn#5]
Congrats on the paper.

I downloaded the code and trained for classification as suggested in the Readme file.
However, upon the test, I could only get a 90.8% classification accuracy.

Please let us know how to achieve 92.1% accuracy on classification as stated in the paper.

Thanks in advance.

**Comment by peterjohnsonhuang** [comment]
Hi, 
The number reported in our paper was trained by adding BatchNorm1d after conv_1, conv_2, conv_3, conv_4 , and the dropout layer, which their corresponding dimension is 64,128,256, 1024, and 256. This version would look like this:
![Screenshot from 2020-10-15 20-48-50](https://user-images.githubusercontent.com/20748448/96128417-ea102580-0f27-11eb-9915-f77d8a028f4a.png)

Note that our minibatch is set as 6. And since there is some randomness in our model, it is better to run the testing several times and vote for the final output. 
Good luck!

**Comment by sheshap** [comment]
@peterjohnsonhuang Thanks for the update. I tried with the above recommendations and I am now able to reproduce 91.9% of classification accuracy. Mostly I will try training from scratch a few more times, then test several times and I am hopeful to reproduce 92.1%.
